src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentEditAction.java

```diff
@@ -97,8 +104,14 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                            + "comment text: " + feedbackResponseComment.commentText.getValue();
         }
 
-        data.comment = feedbackResponseComment;
-
+        // createdAt, lastEditedAt, commentGiverName and lastEditorName are required to generate editedCommentDetails
+        data.comment = updatedComment;
+        data.question = logic.getFeedbackQuestion(response.feedbackQuestionId);
+        data.showCommentToString = StringHelper.toString(updatedComment.showCommentTo, ",");
+        data.showGiverNameToString = StringHelper.toString(updatedComment.showGiverNameTo, ",");
+        data.instructorEmailNameTable = instructorEmailNameTable;
```

Comment:
Good that you went through the thought process 👍 I'd still argue that the added code complexity and reduced readability just to reuse a small part of the `FeedbackResponseCommentRow` isn't worth it. Also, it's possible that in future we might want a different format for the AJAX updated view than in the initial view loaded by the page (determined by `FeedbackResponseCommentRow`) - the two are not really the same.